queries.sql

```diff
@@ -9,3 +9,50 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg <= 17.3 AND weight_kg >= 10.4;
 
+/* transactions */
+BEGIN;
+UPDATE animals
+SET species = 'unspecified'
+WHERE species IS null;
+COMMIT;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE neme LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS null;
+COMMIT;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made%20and%20persists%20after%20commit.), Kindly verify that change was made and persists after commit.